src/__tests__/useObservable.test.ts

```diff
@@ -0,0 +1,108 @@
+import { act, renderHook } from '@testing-library/react-hooks';
```

Comment:
Tests for useObservable were showing some console errors. They are fixed although there are some console errors from other tests that I'll fix on corresponding block of tests.